test/expected/timestamp.out

```diff
@@ -1223,6 +1223,198 @@ FROM unnest(ARRAY[
+        ts_date_trunc('2 days', d, origin => '2020-01-01') as d2,
+        ts_date_trunc('3 days', d, origin => '2020-01-01') as d3,
+        ts_date_trunc('1 week', d, origin => '2020-01-01') as w1,
+        ts_date_trunc('1 week 2 days', d, origin => '2020-01-01') as w1d2
+FROM generate_series('2020-01-01' :: date, '2020-01-12', '1 day') as ts,
+     unnest(array[ts :: date]) as d;
+     d      |     d1     |     d2     |     d3     |     w1     |    w1d2    
+------------+------------+------------+------------+------------+------------
+ 01-01-2020 | 01-01-2000 | 01-01-2000 | 01-01-2000 | 01-01-2000 | 01-01-2000
+ 01-02-2020 | 01-02-2000 | 01-01-2000 | 01-01-2000 | 01-01-2000 | 01-01-2000
+ 01-03-2020 | 01-03-2000 | 01-03-2000 | 01-01-2000 | 01-01-2000 | 01-01-2000
+ 01-04-2020 | 01-04-2000 | 01-03-2000 | 01-04-2000 | 01-01-2000 | 01-01-2000
+ 01-05-2020 | 01-05-2000 | 01-05-2000 | 01-04-2000 | 01-01-2000 | 01-01-2000
+ 01-06-2020 | 01-06-2000 | 01-05-2000 | 01-04-2000 | 01-01-2000 | 01-01-2000
+ 01-07-2020 | 01-07-2000 | 01-07-2000 | 01-07-2000 | 01-01-2000 | 01-01-2000
+ 01-08-2020 | 01-08-2000 | 01-07-2000 | 01-07-2000 | 01-08-2000 | 01-01-2000
+ 01-09-2020 | 01-09-2000 | 01-09-2000 | 01-07-2000 | 01-08-2000 | 01-01-2000
+ 01-10-2020 | 01-10-2000 | 01-09-2000 | 01-10-2000 | 01-08-2000 | 01-10-2000
+ 01-11-2020 | 01-11-2000 | 01-11-2000 | 01-10-2000 | 01-08-2000 | 01-10-2000
+ 01-12-2020 | 01-12-2000 | 01-11-2000 | 01-10-2000 | 01-08-2000 | 01-10-2000
+(12 rows)
+
+-- N month buckets
+SELECT  d,
+        ts_date_trunc('1 month', d) as m1,
+        ts_date_trunc('2 month', d) as m2,
+        ts_date_trunc('3 month', d) as m3,
+        ts_date_trunc('4 month', d) as m4,
+        ts_date_trunc('5 month', d) as m5
+FROM generate_series('2020-01-01' :: date, '2020-12-01', '1 month') as ts,
+     unnest(array[ts :: date]) as d;
+     d      |     m1     |     m2     |     m3     |     m4     |     m5     
+------------+------------+------------+------------+------------+------------
+ 01-01-2020 | 01-01-2020 | 01-01-2020 | 01-01-2020 | 01-01-2020 | 01-01-2020
+ 02-01-2020 | 02-01-2020 | 01-01-2020 | 01-01-2020 | 01-01-2020 | 01-01-2020
+ 03-01-2020 | 03-01-2020 | 03-01-2020 | 01-01-2020 | 01-01-2020 | 01-01-2020
+ 04-01-2020 | 04-01-2020 | 03-01-2020 | 04-01-2020 | 01-01-2020 | 01-01-2020
+ 05-01-2020 | 05-01-2020 | 05-01-2020 | 04-01-2020 | 05-01-2020 | 01-01-2020
+ 06-01-2020 | 06-01-2020 | 05-01-2020 | 04-01-2020 | 05-01-2020 | 06-01-2020
+ 07-01-2020 | 07-01-2020 | 07-01-2020 | 07-01-2020 | 05-01-2020 | 06-01-2020
+ 08-01-2020 | 08-01-2020 | 07-01-2020 | 07-01-2020 | 05-01-2020 | 06-01-2020
+ 09-01-2020 | 09-01-2020 | 09-01-2020 | 07-01-2020 | 09-01-2020 | 06-01-2020
+ 10-01-2020 | 10-01-2020 | 09-01-2020 | 10-01-2020 | 09-01-2020 | 06-01-2020
+ 11-01-2020 | 11-01-2020 | 11-01-2020 | 10-01-2020 | 09-01-2020 | 11-01-2020
+ 12-01-2020 | 12-01-2020 | 11-01-2020 | 10-01-2020 | 09-01-2020 | 11-01-2020
```

Comment:
It would be great if the output format is consistent with the date format we use in the input. I struggle to read `M-d-Y` while `Y-m-d` is much easier to digest and understand.

Reply:
Good point, thanks!

Reply:
This was addressed in the upcoming version of the pull request.